src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/TeamCreation/TeamCreation.styles.ts

```diff
@@ -55,3 +59,50 @@ export const iconButtonCss: CSSObject = {
   width: '2rem',
   marginBottom: '0.5rem',
 };
+
+export const teamCreationModalWrapperCss: CSSObject = {
+  height: '42.5rem',
+  width: '49rem',
+  margin: '1rem',
+  paddingBottom: '5.5rem',
+  maxHeight: 'unset',
+  [tabletMediaQuery]: {
+    height: 'auto',
+    paddingBottom: '0',
+  },
+};
+
+export const teamCreationModalBodyCss: CSSObject = {
+  padding: '0 3.5rem',
+  margin: 'auto 0',
+
+  [tabletMediaQuery]: {
+    padding: '0 1rem',
+    margin: 'auto 0',
+  },
+};
+
+export const confirmLeaveModalWrapperCss: CSSObject = {
+  height: '18.5rem',
+  width: '20rem',
+  margin: '1rem',
+};
+
+export const confirmLeaveModalBodyCss: CSSObject = {
+  padding: '0 1.5rem',
+};
+
+export const confirmLeaveModalButtonsCss: CSSObject = {
+  position: 'absolute',
+  gap: '0.75rem',
+  flexDirection: 'column',
+};
+
+export const buttonCss: CSSObject = {
+  width: '100%',
+  margin: 0,
+  ':disabled': {
+    background: COLOR_V2.GRAY_60,
+    cursor: 'not-allowed',
+  },
```

Comment:
ui-kit buttons should have a disabled state, if not it would be better to address it there.

Colors are always a red flag, they should be a css variable: gray_60 isn't going to work in dark mode (ui-kit should handle this out of the box)

Reply:
https://github.com/wireapp/wire-web-packages/pull/6648